### PR TITLE
I added scheme support for `showAllTables` and `showTablesQuery` for postgresql

### PR DESCRIPTION
**Pull request by okyoky**

### CODE REVIEW

File: lib/dialects/postgres/query-generator.js [hunk 1]
```diff
@@ -102,8 +102,10 @@ var QueryGenerator = {
     });
   },
 
-  showTablesQuery: function() {
-    return "SELECT table_name FROM information_schema.tables WHERE table_schema = 'public' AND table_type LIKE '%TABLE' AND table_name != 'spatial_ref_sys';";
+  showTablesQuery: function(options) {
+    options = options || {};
+    options.schema = options.schema || 'public';
```
Comment (mickhansen) [inline lib/dialects/postgres/query-generator.js]: Have you taken a look at extractTableDetails?

**Comment by mickhansen** [comment]
A test would be great

File: lib/dialects/postgres/query-generator.js (hunk 1 above)
Comment (okyoky) [inline lib/dialects/postgres/query-generator.js]: Hi!

Sorry for my bad english...

I want use sequelize-auto for generate models. sequelize-auto use `showAllTables`.

My next step (after U accept my commit): commit a patch to sequelize-auto for use schema option.

Comment (mickhansen) [inline lib/dialects/postgres/query-generator.js]: Right, but take a look at the extractTableDetails method since that might help you here.

**Comment by janmeier** [comment]
No response  - closing. Feel free to comment if you get time to work more on this @okyoky

**Pull request closed without merging**